lib/matplotlib/transforms.py

```diff
@@ -697,10 +697,11 @@ def union(bboxes):
         """Return a `Bbox` that contains all of the given *bboxes*."""
         if not len(bboxes):
             raise ValueError("'bboxes' cannot be empty")
-        x0 = np.min([bbox.xmin for bbox in bboxes])
-        x1 = np.max([bbox.xmax for bbox in bboxes])
-        y0 = np.min([bbox.ymin for bbox in bboxes])
-        y1 = np.max([bbox.ymax for bbox in bboxes])
+        with np.errstate(invalid='ignore'):
```

Comment:
add a reference for the relevant numpy versions so that this can be removed once we don't support these numpys anymore?